### PR TITLE
Dev/202: TestcaseIOsPatching

### PR DESCRIPTION
**Pull request by Johnny850807**
resolve: #202

### CODE REVIEW

**Review by Wally5077 — CHANGES_REQUESTED** [review]
review done

File: src/app/problem/editor/right/testcases/TestcaseEditor.js [hunk 1]
```diff
@@ -48,58 +47,72 @@ const BulletText = ({title, value, onChange, isEditing, type = "text"}) => {
...
 
-const UploadFileItems = ({buttonName, buttonColor, files, addFile, removeFile, isEditing}) => {
+const UploadFileItems = ({buttonName, buttonColor, files, onFilesUploaded, removeFile, isEditing, multipleFiles, fileRemovable = true}) => {
     return (
         isEditing ?
             <>
-                <FixedUploadFileItems files={files} removeFile={removeFile} style={{marginBottom: "5px"}}/>
+                <FixedUploadFileItems items={files.map(f => {
+                    return {key: f.name, text: f.name}
+                })} fileRemovable={fileRemovable} removeItem={item => removeFile(item.text)}
```
[the remark this answers is not in the document]
Comment (Johnny850807) [reply src/app/problem/editor/right/testcases/TestcaseEditor.js:67]: It cannot be achieved. 

The first pair of braces will be seen as the wrapper of multi-line statements within the lambda.

[the remark this answers is not in the document]
Comment (Johnny850807) [reply src/app/problem/editor/right/testcases/TestcaseEditor.js:67]: s

**Comment by Johnny850807** [comment]
asdas

File: src/app/problem/editor/right/testcases/TestcaseEditor.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Wally5077) [reply src/app/problem/editor/right/testcases/TestcaseEditor.js:67]: ok

**Review by Wally5077 — APPROVED** [review]
LGTM

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**